google-cloud-clients/google-cloud-dlp/pom.xml

```diff
@@ -15,7 +15,7 @@
     <version>0.47.1-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-clients:current} -->
   </parent>
   <properties>
-    <site.installationModule>google-cloud-dlp-v2beta1</site.installationModule>
+    <site.installationModule>google-cloud-dlp-v2</site.installationModule>
```

Comment:
wait this doesn't seem right - shouldn't this match the artifact? (`google-cloud-dlp`)